### DISCUSSION TITLE
How to turn off tracking?

### DISCUSSION THREAD

**Discussion by etlweather** [voiceip/oreka#119] [Q&A]
From what I remember, in a previous version, I was able to turn off the calls to orktrack - but I just compiled the latest and it keeps trying to contact orktrack which I am not running and I can't find how to turn that feature off. 

Is it possible? While it does not break anything, it is a bit annoying to constantly see the failed calls in the log.

**Comment by etlweather** [comment]
I guess this communication channel is not monitored

**Reply by kingster** [reply]
Hi @etlweather

Sorry, somehow missed this thread. Since the time I have been using orkaudio I have never seen/used this option. 

But just asking did you try setting the configuration to empty strings?

**Reply by etlweather** [reply]
@kingster No problem. Well yesterday I think I found a more fundamental problem. It does not look like the config file is being used. I added some garbage characters (to cause invalid XML) and that didn't even cause any issue. 

I am running on `arm`, built from master branch. I also tried one or two previous release tags. The config file is the one created by the `make install` command in `/etc/orkaudio/config.xml`

But I swear earlier I made a change to that config file that loaded, now my changes don't seem to be loading.

**Reply by kingster** [reply]
Hi

Just tried this

```
I added some garbage characters (to cause invalid XML) and that didn't even cause any issue.
```

Observed same behaviour, doesn't cause issues.

```xml
cat /etc/orkaudio/config.xml
<config>
aaaa
	<AudioOutputPath>/var/log/orkaudio/audio</AudioOutputPath>
	<CapturePlugin>libvoip.soa</CapturePlugin>
...
```

However, I can see the configs are read properly.


```
2022-10-17 22:24:01,993 ERROR root:101 - Failed to load the following plugin: /usr/lib/libvoip.soa /usr/lib/libvoip.soa: cannot open shared object file: No such file or directory, could be missing dependency, Try running in cmd box (orkaudio debug)
```

Also, checking documentation, saw this `<EnableReporting>true</EnableReporting>` which can be set to false to turn off sending it to tracker. Please try it out.

**Reply by etlweather** [reply]
Alright, some config do affect orkaudio - I changed the format to pcmwav and file path and that worked.

I tried `EnableReporting` and that didn't change anything. Or maybe it did. I get an error in the log on connecting to orktrack at start up but I don't seem to get further error logs when a phone call occurs so that is probably fine.

I set the interface to monitor and it does pick up from those two interface but it still seem to do * because in the logs, it logs all interfaces as being monitored - not really a problem, just interesting/weird.

**Reply by etlweather** [reply]
Actually, I still see routine errors calling orktrack. I guess it does not really matter if it does not crash the program.
